upgrade/upgrading-monorepo.md

```diff
@@ -13,8 +13,6 @@ Typical upgrade sequence should be:
 
 ***Note:** During the execution of `build-demo-dev phing target`, there will be installed 3-rd party software as dependencies of Shopsys Framework by [composer](https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies) and [npm](https://docs.npmjs.com/about-the-public-npm-registry) with licenses that are described in document [Open Source License Acknowledgements and Third-Party Copyrights](https://github.com/shopsys/shopsys/blob/7.3/open-source-license-acknowledgements-and-third-party-copyrights.md)*
 
-## [From v10.0.0 to 11.0.0-dev]
```

Comment:
I would say this should be actually replaced with
```
## [From v10.0.2 to 10.0.3-dev]

## [From v10.0.0 to v10.0.2]

Reply:
I don't know... These empty sections seem kinda useless to me...